ci/test_install.sh

```diff
@@ -7,3 +7,10 @@ docker run -i --rm\
   --entrypoint /bin/bash\
   -w /tmp\
   python:3.9 /ci/smoketest.sh
+
+docker run -i --rm\
+  -v $PWD/dist:/dist -v $PWD/appmap/test/data/unittest:/appmap/test/data/unittest\
+  -v $PWD/ci:/ci\
+  --entrypoint /bin/bash\
+  -w /tmp\
+  python:3.9 /ci/test_pipenv.sh
```

Comment:
Perhaps these scripts could be ran in a single `docker run` instance.

Reply:
Ok, it should be straightforward to combine them.